### PR TITLE
homogeneous visual coherence for messages and alignment of footer boxes

### PR DESCRIPTION
**Pull request by GusBaamonde**
message_reports

### CODE REVIEW

File: assets/scss/templates/_reportes.scss [hunk 1]
```diff
@@ -1,13 +1,14 @@
 
 .dp-wrapper-messages {
-  max-width: $dp-wrapper-max-width;
+  max-width: 960px; /* check to modularize the messages */
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: Please change it in order to search it easily:

```suggestion
  max-width: 960px; /* TODO: check to modularize the messages */
```

**Comment by andresmoschini** [comment]
It looks fine, did you verify how it affects other pages? 

@santiagopaolucci agree on these changes?

**Comment by andresmoschini** [comment]
Do not forget to rebase it.

**Comment by GusBaamonde** [comment]
> It looks fine, did you verify how it affects other pages?

If I verify ;)

**Comment by GusBaamonde** [comment]
> Do not forget to rebase it.

I have to go to webapp

**Pull request merged**